src/SourceDialog/SourceDialog.vala

```diff
@@ -209,7 +209,7 @@ public class Maya.View.SourceDialog : Gtk.Grid {
         });
 
         color_button_orange.toggled.connect (() => {
-            hex_color = "#f37329";
+            hex_color = "#ffa154";
```

Comment:
This adjustment only provides consistency. Unfortunately I cannot yet speak of WCAG compliance here. For this we would need a background of at least @ORANGE_700 with an assumed foreground color of #fff.